src/graph/backend/dnnl/dnnl_op_def.hpp

```diff
@@ -988,6 +988,8 @@ DNNL_GRAPH_OP_SCHEMA(dnnl_softmax, 1,
                 .set_output(1, "scratchpad")
                 // Attributes inherited from SoftMax
                 .set_attr(op_attr::axis, false, attribute_kind::i, (int64_t)1)
+                .set_attr(op_attr::mode, false, attribute_kind::s, "none",
+                        {"none", "inf_as_zero"})
```

Comment:
Do you need to add this mode attr to internal dnnl_sdpa op also?

Reply:
Yes, i guess so. I will add it along with GPU ukernel enabling. We will need a way to pass the flag along to the kernel in backend.